frontend/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/youtube-player": "^12.1.3",
     "@auth0/angular-jwt": "^5.0.2",
     "@microsoft/signalr": "^5.0.8",
+    "angular-ng-autocomplete": "^2.0.5",
```

Comment:
Why do you use it instead of the PrimeNg version?

Reply:
I have deleted it)